### ISSUE TITLE
🐛 Panic `Malformed list, node expected but found token...` in `biome_rowan/src/ast/mod.rs`

### ISSUE THREAD

**Issue by omasakun** [biomejs/biome#7101]
### Environment information

```block
CLI:
  Version:                      2.1.3
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_PATH:               unset
  BIOME_LOG_PREFIX_NAME:        unset
  BIOME_CONFIG_PATH:            unset
  BIOME_THREADS:                unset
  NO_COLOR:                     unset
  TERM:                         xterm-256color
  JS_RUNTIME_VERSION:           v22.15.0
  JS_RUNTIME_NAME:              node
  NODE_PACKAGE_MANAGER:         pnpm/10.13.1

Biome Configuration:
  Status:                       Not set
  Path:                         unset

Workspace:
  Open Documents:               0
```

### What happened?

1. Create a typescript file `index.ts`

```ts
export class Foo {
  constructor(private a: number, private b: number) {}
}
```

2. Run `npx @biomejs/biome@2.1.3 lint --fix --unsafe`

Result:

```
Biome encountered an unexpected error

This is a bug in Biome, not an error in your code, and we would appreciate it if you could report it to https://github.com/biomejs/biome/issues/ along with the following information to help us fixing the issue.

When opening the issue, please provide a minimal reproduction, or identify and share the file/code that triggers it. Without a way to reproduce the error, the error can't be fixed:

Source Location: /__w/biome/biome/crates/biome_rowan/src/ast/mod.rs:662:41
Thread Name: biome::workspace_worker_2
Message: Malformed list, node expected but found token COMMA@33..35 "," [] [Whitespace(" ")] instead. You must add missing markers for missing elements.

index.ts internalError/panic  INTERNAL  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━

  ✖ processing panicked: Malformed list, node expected but found token COMMA@33..35 "," [] [Whitespace(" ")] instead. You must add missing markers for missing elements.
  
  ⚠ This diagnostic was derived from an internal Biome error. Potential bug, please report it if necessary.
  

Checked 1 file in 6ms. No fixes applied.
```

Playground: https://biomejs.dev/playground/?analyzerFixMode=safeAndUnsafeFixes&code=ZQB4AHAAbwByAHQAIABjAGwAYQBzAHMAIABGAG8AbwAgAHsACgAgACAAYwBvAG4AcwB0AHIAdQBjAHQAbwByACgAcAByAGkAdgBhAHQAZQAgAGEAOgAgAG4AdQBtAGIAZQByACwAIABwAHIAaQB2AGEAdABlACAAYgA6ACAAbgB1AG0AYgBlAHIAKQAgAHsAfQAKAH0A

### Expected result

It should not throw an error.

### Code of Conduct

- [x] I agree to follow Biome's Code of Conduct

**Comment by dyc3** [comment]
I'm not seeing the same diagnostic as you are showing here in the playground

**Comment by omasakun** [comment]
When I open the Analyzer Fixes tab, it says “Can't apply fixes with errors,” and when I open the DevTools console, an error is displayed.

**Comment by omasakun** [comment]
I think there is an error in the handling of properties defined in the constructor arguments.

For example, when the following code is fixed with Safe and Unsafe,

```ts
export class Foo {
  constructor(private a: number) {
    console.log(a)
  }
}
```

The result is

```ts
export class Foo {
  constructor() {
    console.log(a)
  }
}
```

But, it should be

```ts
export class Foo {
  constructor(a: number) {
    console.log(a)
  }
}
```

Playground: https://biomejs.dev/playground/?analyzerFixMode=safeAndUnsafeFixes&code=ZQB4AHAAbwByAHQAIABjAGwAYQBzAHMAIABGAG8AbwAgAHsACgAJAGMAbwBuAHMAdAByAHUAYwB0AG8AcgAoAHAAcgBpAHYAYQB0AGUAIABhADoAIABuAHUAbQBiAGUAcgApACAAewAKACAAIAAgACAAIAAgAGMAbwBuAHMAbwBsAGUALgBsAG8AZwAoAGEAKQAKACAAIAAgACAAfQAKAH0ACgA%3D

(Should I create a separate issue for this?)

**Comment by omasakun** [comment]
I'm not familiar with the Biome code base, so I may be wrong, but it seems that this part of the code does not take into account that class members can be defined in the constructor arguments.

https://github.com/biomejs/biome/blob/f0e36e223bd0784a5846a29a52c5550252cb12bb/crates/biome_js_analyze/src/lint/correctness/no_unused_private_class_members.rs#L109

**Comment by dyc3** [comment]
Thanks for clarifying, I can see it in the console.
